### PR TITLE
SMILE-1239: fix role policies

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -347,6 +337,19 @@ Resources:
...
+    Properties:
+      PolicyName: !Sub DefaultLambdaPermissions-${AWS::StackName}
+      Roles:
+        - !Ref ReadFromScopusBucketsRole
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - 's3:*'
```
Comment (nraanes) [inline template.yaml:351]: `- s3:GetObject` is sufficient for NvaScopusFunction I believe

File: template.yaml [hunk 2]
```diff
@@ -347,6 +337,19 @@ Resources:
...
+            Resource: "*"
```
Comment (nraanes) [inline template.yaml:352]: Should we set the specific bucket or is that not necessary? (ScopusXmlBucket)

File: metadata-transformer/src/main/java/no/unit/nva/metadata/CreatePublicationRequest.java [hunk 3]
```diff
@@ -21,6 +23,8 @@
     private JsonNode context;
     private List<ResearchProject> projects;
     private List<URI> subjects;
+    private Set<AdditionalIdentifier> additionalIdentifiers;
+
```
Comment (gregorgabriel) [inline metadata-transformer/src/main/java/no/unit/nva/metadata/CreatePublicationRequest.java:27]: Denne er vel ikke del av role policies? Den passer kanskje mer til Anette sin forespørsel om hvor vi legger doi, pmid osv. til.

**Review by gregorgabriel — APPROVED** [review]

**Review by anetteOlli — APPROVED** [review]

File: template.yaml (hunk 1 above)
Comment (axthosarouris) [reply template.yaml:351]: I will restrict the access after we have established that the handler works.

**Pull request merged**